collector/src/output_frontend.h

```diff
@@ -120,15 +141,18 @@ class CMonitorOutputFrontend {
 private:
     class CMonitorOutputMeasurement {
     public:
-        CMonitorOutputMeasurement(const char* name = "", const char* value = "", bool numeric = false)
+        CMonitorOutputMeasurement(
+            const char* name = "", const char* value = "", double dvalue = 0, bool numeric = false)
```

Comment:
since we're changing the logic here to store the value also as a double, I think we should split this ctor in two overloads:

```
        CMonitorOutputMeasurement(const char* name = "", const char* value = "")
        {
            strncpy(m_name.data(), name, CMONITOR_MEASUREMENT_NAME_MAXLEN - 1);
            strncpy(m_value.data(), value, CMONITOR_MEASUREMENT_VALUE_MAXLEN - 1);
            m_dvalue = 0;
            m_numeric = false;
        }
        CMonitorOutputMeasurement(const char* name, double value)
        {
            strncpy(m_name.data(), name, CMONITOR_MEASUREMENT_NAME_MAXLEN - 1);
            m_value[0]='\0';
            m_dvalue = value;
            m_numeric = true;
        }
```

Reply:
this way ALL string kpis must be forced to use first ctor; all numeric kpis must be forced to use the 2nd ctor

Reply:
also note htat CMonitorOutputFrontend::phex() is unused for now and should probably be removed